e_os.h

```diff
@@ -295,6 +295,64 @@ struct servent *getservbyname(const char *name, const char *proto);
 # endif
 /* end vxworks */
 
+/* ----------------------------- HP NonStop -------------------------------- */
+/* Required to support platform variant without getpid() and pid_t. */
+#ifdef __TANDEM
+#  include <strings.h>
+#  include <netdb.h>
+#  define getservbyname(name,proto)          getservbyname((char*)name,proto)
+#  define gethostbyname(name)                gethostbyname((char*)name)
+#  define ioctlsocket(a,b,c)	ioctl(a,b,c)
+#  ifdef NO_GETPID
+inline int nssgetpid();
+#    ifndef NSSGETPID_MACRO
+#      define NSSGETPID_MACRO
+#      include <cextdecs.h(PROCESSHANDLE_GETMINE_)>
+#      include <cextdecs.h(PROCESSHANDLE_DECOMPOSE_)>
+       inline int nssgetpid()
+       {
+         short phandle[10]={0};
+         union pseudo_pid {
+          struct {
+           short cpu;
+           short pin;
+         } cpu_pin ;
+         int ppid;
+        } ppid = { 0 };
+        PROCESSHANDLE_GETMINE_(phandle);
+        PROCESSHANDLE_DECOMPOSE_(phandle, &ppid.cpu_pin.cpu, &ppid.cpu_pin.pin);
+        return ppid.ppid;
+       }
+#    endif /* NSSGETPID_MACRO */
+#    define getpid(a) nssgetpid(a)
```

Comment:
These two lines should trade places

Reply:
> 
> 
> These two lines should trade places

I'm not sure about that. The `nssgetpid(a)` is defined outside the `NSSGETPID_MACRO` block. I'm actually not sure why the person who did this did that. But I'll go with your change until I get yelled at.

Reply:
Er, no. It's defined inside that block, not outside. It may be declared outside of it, but that's unnecessary.